Adding information about launching on iOS Device from menu (#678)
spatialos/gdk-for-unity: docs/content/mobile/ios/setup.md

    * [Xcode](https://developer.apple.com/xcode/)
      * Once installed, open Xcode. It might prompt you to accept user agreements.
    * (Optional) [Unity Remote](https://itunes.apple.com/gb/app/unity-remote-5/id871767552?mt=8) - this is Unity’s solution for faster development iteration times.

>     * (Optional) [ideviceinstaller](https://github.com/libimobiledevice/ideviceinstaller) and [idevicedebug](https://helpmanual.io/help/idevicedebug/) - iOS device command line interaction tools. They will allow you to connect to local deployment directly from Unity.

Reviewer: ```suggestion
    * (Optional) [ideviceinstaller](https://github.com/libimobiledevice/ideviceinstaller) and [idevicedebug](https://helpmanual.io/help/idevicedebug/) - iOS device command line interaction tools. Use this to connect to local deployment directly from Unity.
```
Make it more active is better and clearer.
